.github/workflows/production.yml

```diff
@@ -0,0 +1,49 @@
+name: Production Deployment
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  # This job will:
+  #   * deploy a production build every time there is a push on the master branch
+  #   * comment that commit with the deploy URL
+  publishMasterCommit:
+    name: Publish to Netlify
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      # Checkout to our latest code on current branch
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # Install NodeJS
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Test linting
+        run: yarn lint  
+
+      - name: Run test cases
+        run: yarn ci:test 
+        
+      - name: Create build
+        run: yarn run build
+
+      # Deploy to Netlify
+      - name: Deploy production to Netlify
+        uses: South-Paw/action-netlify-deploy@v1.2.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          netlify-auth-token: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          netlify-site-id: ${{ secrets.NETLIFY_SITE_ID }}
```

Comment:
yes, actually draft deployment gives a temp URL that is not published on our main Netlify URL. production is published once we merge the PR so it will publish the master to our main Netlify site.

Reply:
This works for now, if we go ahead with staging later, we can modify this.